### ISSUE TITLE
Error 404 for frontend requests (only in certain environment)

### ISSUE THREAD

**Issue by codenius** [DoctorFuchs/kiosk#19]
Can't access html page (showing not found error), instead getting **error 404 for every request for frontend resources**, but **not for backend/api**.
Appears only on **Linux Mint 19.3 32 bit** machine, **Python 3.6.9.** .
Checkout the log below.

```
benutzer@benutzer:~/Schreibtisch/kiosk$ python3 run.py -b
 * Running on http://localhost:1024/ (Press CTRL+C to quit)
127.0.0.1 - - [02/Jan/2022 15:42:22] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:42:23] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:42:54] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:42:54] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:42:54] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:43:16] "GET /frontend HTTP/1.1" 404 -
127.0.0.1 - - [02/Jan/2022 15:43:29] "GET /frontend/index.html HTTP/1.1" 404 -
```

**Comment by codenius** [comment]
Probably fixed trough 54d876e. Final tests still pending.

**Comment by DoctorFuchs** [comment]
Fixed, i guess
